app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = []
+    for person in people:
+        human = Person(person["name"], person["age"])
+        if person.get("wife"):
+            for human2 in people:
+                if person["wife"] == human2["name"]:
+                    person_wife = Person(human2["name"], human2["age"])
```

Comment:
I believe you will create x2 instances of the Person class with this approach. Try to do this task in two steps:

create all people without partners
fill partners with existing instances (use Person.people list to find a proper partner)